strategies/attestationdata/combinedmajority/attestationdata.go

```diff
@@ -370,6 +395,18 @@ func (*Service) attestationDataLoop2(ctx context.Context,
 		Msg("Results")
 }
 
-func generateAttestationKey(attestationData *phase0.AttestationData) string {
-	return fmt.Sprintf("%s-%s-%d", attestationData.Source.Root.String(), attestationData.Target.Root.String(), attestationData.Slot)
+func generateattestationDataHead(attestationData *phase0.AttestationData) (attestationDataHead, error) {
+	sourceRoot, err := attestationData.Source.HashTreeRoot()
```

Comment:
Do we need a HashTreeRoot for these or can we just use the raw `phase0.Checkpoint`?